content/_data/changelogs/weekly.yml

```diff
@@ -8542,6 +8542,76 @@
 
   # pull: 4875 (PR title: Add new line at EOF in config.xml)
 
+- version: '2.251'
+  date: 2020-08-04
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4870
+    issue: 63180
+    authors:
+    - fqueiruga
+    message: |-
+      Restore wrapping tabs into multiple lines instead of overflowing (regression in 2.248).
+  - type: bug
+    category: bug
+    pull: 4893
+    issue: 63232
+    authors:
+    - Wadeck
+    message: |-
+      Show build time data in the Build Time Trend Page (regression in 2.245).
+  - type: bug
+    category: bug
+    references:
+      - url: https://github.com/jenkinsci/dark-theme-plugin/issues/126
+        title: Fixes bread crumbs flash in Dark Theme
+    issue: 63285
+    pull: 4867
+    authors:
+    - fqueiruga
+    message: |-
+      Normalize widget colors to be consistent with the new color palette.
+  - type: bug
+    category: regression
+    pull: 4892
+    issue: 63276
+    authors:
+    - fqueiruga
+    message: |-
+      Empty installed plugins table text is readable again (regression in 2.249).
+  - type: bug
+    category: bug
+    pull: 4863
+    issue: 63166
+    authors:
+    - takashiharano
+    message: |-
+      Replace text references to slave with agent in Japanese documentation and messages.
+  - type: bug
+    category: bug
+    pull: 4889
+    issue: 42228
+    authors:
+    - takashiharano
+    message: |-
+      Prevent JavaScript error when registering validators in some cases.
+  - type: rfe
+    category: developer
+    pull: 4825
+    issue: 62757
+    authors:
+    - stellargo
+    message: |-
+      Developer: Allow migration of fingerprints from local storage to external storage.
+
+  # pull: 4865 (PR title: Align plugin manager filter with table)
+  # pull: 4869 (PR title: Fix typo in Lithuanian translation.)
+  # pull: 4871 (PR title: Log expected and actual values for better UTF-8 decoding diagnosability)
+  # pull: 4876 (PR title: Fix search JavaScript issue)
+  # pull: 4877 (PR title: Update at-since Javadoc)
+  # pull: 4881 (PR title: Do not block rendering of Manage Jenkins while waiting for UC data)
```

Comment:
I would still changelog it

Reply:
OK.  I thought that the [note](https://github.com/jenkinsci/jenkins/pull/4881) from @daniel-beck that said:

> (Too minor)

was the hint that it should not be in the changelog.

Added in 33007d343ed9a89dbea3b2bb35db76b3701e479f.  Since the release has been delivered, I plan to merge as soon as CI passes.